app.js

```diff
@@ -157,35 +158,83 @@ lookAndFeel.configure(app, {
   }
 });
 
-journey(app, {
-  baseUrl,
-  steps,
-  session: {
-    redis: {
-      url: config.redis.url,
-      connect_timeout: 15000
+function isTest() {
+  return process.env.NODE_ENV === 'mocha';
+}
+
+
+if (isTest()) {
```

Comment:
I think we can put this into pa11y's own app.js. It's been done for TYA I believe and ES so should 't be too difficult. I'll have a look when I have time.